docs/docs/KerasStyleAPIGuide/Layers/activation.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Delete this line?

Reply:
@jason-dai I fixed the style problem. It now shows correctly rendered by GitHub :)